journald/lib_journald.hpp

```diff
@@ -80,6 +83,13 @@ struct LoggerFlags : public virtual flags::FlagsBase
         "    size <logrotate_max_stderr_size>\n"
         "  }\n"
         "NOTE: The 'size' option will be overridden by this module.");
+
+    add(&LoggerFlags::extra_labels,
+        "extra_labels",
+        "Extra kay value pairs (in JSON object format) that will be set\n"
+        "for the journald entry. For example:\n"
```

Comment:
Do we need to point out that the  values can only be strings, not numbers or other objects?

Reply:
Maybe you could also mention that this field is meant for cases where ExecutorInfo labels are not applicable, like standalone containers.

Reply:
Done.